cwatch.php

```diff
@@ -416,6 +516,21 @@ private function pushUserAndLicenses($vars, array $service_licenses = null)
                     $license_keys[] = $license->distributionResult[0]->licenseKeys[0];
                 }
             }
+
+            if (!empty($vars['cwatch_domain']) && $package->meta->cwatch_package_type == 'single_license') {
+                // Add the domain submitted for a single license package
+                $site_response = $api->addSite(
+                    [
+                        'email' => $vars['cwatch_email'],
+                        'domain' => $vars['cwatch_domain'],
+                        'licenseKey' => $license_keys[0],
```

Comment:
Looks like you presume there is always a 0th index set on the $license_keys, but that may not be the case depending on what is set for the $license_types and whether there is a successful license API response